plugins/dynamic-dropdown/README.md

```diff
@@ -0,0 +1,89 @@
+# Builder.io dynamic dropdown
+
+See [here](src/dropdown.tsx) for the React component that powers this plugin
```

Comment:
I should have mentioned sooner, but would be great to describe what this plugin does/why and how to use it (e.g. with simple example here)